umodbus/common.py

```diff
@@ -81,6 +80,7 @@ def send_response(self,
         :param      signed:  Indicates if signed values are used
         :type       signed:  bool
         """
+        print("sending sync response...", type(self).__name__, "->", type(self._itf).__name__)
```

Comment:
this is either a logging output or can be removed

Reply:
Yeah this was mainly for debugging. Removed, thanks!